components/Aduan/DetailAduan/TextTitle.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <span
+    class="text-gray-700 dark:text-dark-text-low text-[12px] font-lato font-normal"
+  ><slot /></span>
+</template>
```

Comment:
Discussion
Also here, if we want to show the title, we can use the heading tag `h1-h6` because it's more semantic.

WDYT?

references:
1. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
i changes tags [here](https://github.com/jabardigitalservice/super-app-web/pull/63/commits/fa6235acb6320da5348940d2e6981ebada0b06fb)